src/variables.js

```diff
@@ -14,37 +82,78 @@ const Users = [
     name: 'First Last',
     initials: 'FL',
     color: getCSSVar('--purpleIcon'),
-    class: '202X',
+    class: 2023,
     school: ['A School'],
     major: ['Some Interesting Major', 'Some Other Major'],
     bio: 'Really interesting things about my background, interests, hobbies, etc.',
     posts: [],
-    saved: []
+    saved: [],
+    display: []
   },
   {
     name: 'John Doe',
     initials: 'JD',
     color: getCSSVar('--pinkIcon'),
-    class: '2025',
+    class: 2025,
     school: ['Bienen, Weinberg'],
     major: ['Trumpet Performance', 'Psychology'],
     bio: 'I like touching grass.',
     posts: [],
-    saved: []
+    saved: [],
+    display: []
   },
   {
     name: 'Some Person',
     initials: 'SP',
     color: getCSSVar('--yellowIcon'),
-    class: '2027',
+    class: 2027,
     school: ['McCormick'],
     major: ['Mechanical Engineering'],
     bio: 'I don\'t like touching grass.',
     posts: [],
-    saved: []
+    saved: [],
+    display: []
   }
 ]
 
+// fetch users from database and add them to User
+async function getAllUsers() {
+  try {
+    const response = await fetch("https://disc-assignment-5-users-api.onrender.com/api/users");
+    if (!response.ok) {
+        throw new Error(`Error: ${response.status} - ${response.statusText}`);
+    }
+
+    const fetchedUsers = await response.json();
+    
+    // Map over fetched users to add missing properties with default values
+    const newUsers = fetchedUsers.map(user => ({
+        name: `${user.firstName} ${user.lastName}`,
+        initials: `${user.firstName[0]}${user.lastName[0]}`,
+        color: getCSSVar('--lightGray'),
+        class: user.graduationYear,
+        school: 'NA',
+        major: user.major,
+        bio: user.bio,
+        posts: [],
+        saved: [],
+        display: []
+    }))
+
+    // Add new users to the Users array
+    Users.push(...newUsers);
+
+    console.log("Updated Users array:", Users);
+    return Users;
+  } 
```

Comment:
Good use of try-catch here! Always assume that anything you await can throw errors, especially API calls. In general, this is a well written API call function, so good job on that!